add input file example with 1M line (#5)
geeksblabla/blanat: README.md

### Constraints:

- 1.00 <= price <= 100.00
- we have 101 cities.
- We have 100 products.
- Each city may have the same product duplicated several times with different or identical prices.
- Your code will process a 10^9 (1 billion) line CSV file.
- You are allowed to code with any language from this list: C, C++, Java, Python, Rust, JavaScript, PHP.
- The input file is located on "input.txt" 

> - The output file should be located in the same fodler directory as your submission `./output.txt`

Reviewer: @aboullaite what does this mean?

the output file should mean where the output file will be written by the solution, and it should write the result to a file called `output.txt`. This makes it sound that you have to submit the output file as well, which I don't think what we want

Author: How about ?
```suggestion
- Make sure the result are saved in a file saved under `./output.txt`
```

Reviewer: yeah looks better (we can even provide a dummy solution that reads from `input.txt` and prints to `output.txt`)

### INPUT

> You can find an example of the input file [here](./input.txt). This file contains 1 million line to run your tests and validate your solution.

Author: @chermehdi What is the expected file location ? same for output file if any

Reviewer: Output should be a file named `output.txt` in the working directory of the process

> You can find an example of the input file [here](./input.txt). This file contains 1 million line to run your tests and validate your solution.

Reviewer: Also, would've been nice to give people the expected output as well.

Author: Once we agreed on the program I can run it and share the output for the 1M rows
